db/models.py

```diff
@@ -17,7 +21,7 @@ def __str__(self) -> str:
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
There is no need to make `title` a `db_index` field. Remove this change here and remake migration to encounter that.